src/AuthForm.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import {
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import { auth } from "./firebase";
+import "./App.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Form from "react-bootstrap/Form";
+
+class AuthForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      emailInputValue: "",
+      passwordInputValue: "",
+      isNewUser: true,
+      errorCode: "",
+      errorMessage: "",
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    const closeAuthForm = () => {
+      this.setState({
+        emailInputValue: "",
+        passwordInputValue: "",
+        isNewUser: true,
+        errorCode: "",
+        errorMessage: "",
+      });
+      this.props.toggleAuthForm();
+    };
```

Comment:
Maybe it would make sense to define this functions outside of this method, otherwise they get redefined on every form submit